### PR TITLE
Update descriptions for `TriggerTemplates` and `TriggerBindings`

### PR DESCRIPTION
**Pull request by sergetron**
# Changes

Updates descriptions for `TriggerTemplates` and `TriggerBindings` to correct technical inaccuracies.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [X] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

<!--
Describe any user facing changes here, or delete this block.

Examples of user facing changes:
- API changes
- Bug fixes
- Any changes in behavior
- Changes requiring upgrade notices or deprecation warnings

For pull requests with a release note:

```release-note
NONE
```

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -26,27 +26,27 @@ cluster as an extension to Tekton Pipelines.
 Tekton Triggers consists of a controller service that runs on your Kubernetes cluster as well as the following Kubernetes Custom Resource Definitions (CRDs) that extend
 the functionality of Tekton Pipelines to support events:
 
-*  [`EventListener`](https://github.com/tektoncd/triggers/blob/main/docs/eventlisteners.md) - listens for events at a specified port on your Kubernetes cluster.
+*  [`EventListener`](https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md) - listens for events at a specified port on your Kubernetes cluster.
```
Comment (dibyom) [inline docs/README.md:29]: We should revert these. The default branch now is main.

Comment (sergetron) [reply docs/README.md]: Updated all Triggers docs links from `master` to `main`.

**Review by dibyom — APPROVED** [review]
/lgtm

**Pull request merged**